templates/ServiceNow/ServiceNowCase.html

```diff
@@ -0,0 +1,49 @@
+<div class="coveo-result-frame">
+  <div class="coveo-result-cell" style="vertical-align: top;padding-left: 16px;">
+    <div class="coveo-result-row" style="margin-top:0;">
+      <div class="coveo-result-cell" style="vertical-align:top; font-size:16px">
+        <a class="CoveoResultLink" data-title-template="[${raw.snnumber}] ${raw.title} ">
+        </a>
+      </div>
+      <div class="coveo-result-cell" style="width:120px">
+        <span class="CoveoFieldValue" data-field="@date" data-text-caption="Updated: " data-helper="dateTime" data-html-value="false"
+          data-helper-options-include-time-if-this-week="false" style="color:#6A7689;font-size:10px"></span>
+      </div>
+    </div>
+    <div class="coveo-result-row" style="margin-top:10px;">
+      <div class="coveo-result-cell">
+        <span style="border-style:solid;border-width:thin;border-color:#3A6692;border-radius:0px 10px 10px 0px;font-size:12px;background-color:#3A6692;color:white;padding:2px 10px 0 10px;margin-right:-12px">CSM Case</span>
+        <span class="CoveoFieldValue" data-field="@snstate" style="border:solid thin #3A6692;border-radius:0px 10px 10px 0px;font-size:12px;color:black;padding-left:15px;padding-right:10px;padding-top:2px;padding-bottom:2px"></span>
+        <span class="CoveoFieldValue" data-field="@snproductname" data-text-caption="Product " style="padding-left:10px"></span>
+        <span class="CoveoFieldValue" data-field="@snaccountname" data-text-caption="Account " style="padding-left:10px"></span>
+      </div>
+      <div class="coveo-result-cell" style="width:120px">
+        <span class="CoveoQuickview" style="font-size:10px;color:black;border-width:thin;border-color:grey;border-style:solid;border-radius:16px;width:100px;padding-right:10px;padding-left:10px;padding-top:2px;padding-bottom:2px;text-decoration:none">QUICK VIEW</span>
```

Comment:
Any reason this seems very different from all other templates ? 

Eg: the Quickview for all other types of results is just a `<div class="CoveoQuickview"></div>` and Service now would look different from all the rest.

Is this really wanted ? Wouldn't that look silly in an actual page where you have content from multiple source ?

This comment is not only for the `Quickview` part, but for everything that is kind of "standard" in other templates. Eg: "Excerpt", or border around a lot of stuff.


Would you also mind pasting a screenshot of what it looks like in this PR ?

Reply:
We worked with the design team and they came up with those.

Reply:
Ok, but, was it clear on their side that it's different from the rest of the templates ? 
Or was it worked on a "silo" kind of way ? :)

I mean, I'm fine if that's a deliberate decision, but won't it look silly when you display content from multiple source inside ServiceNow ?

Reply:
Yeah, our current templates would probably need a bit of polish to match this new design, but I find it way better.